t/venom/pfservers/configurator/step0.yml

```diff
@@ -0,0 +1,16 @@
+name: Step 0 preconfiguration
+version: 2
+testcases:
+- name: set_hostname
+  steps:
+  - script: 'hostnamectl set-hostname {{.configurator_hostmame_machine}}'
+
+- name: set_gateway
+  steps: 
+  - script: 'route add default gw {{.configurator_gateway}}'
+
+- name: restart_network
+  steps:
+  - script: 'systemctl restart network'
+
+
```

Comment:
I don't understand the need of this file. All these elements can be configured using configurator API.

Could you include all these elements in the step1 ?

Reply:
This step is for preparing the machine. But if this configuration is done by ansible and not Venom then I will just remove this step.